packages/pn-personafisica-login/.env.development.local

```diff
@@ -1,15 +1,11 @@
+
```

Comment:
Questo file non dovrebbe essere nel repository remoto ma solo in locale. Chiedere a Carlotta per conferma

Reply:
@AndreaCimini90 confermo, è stato un mio refuso quando ho aggiunto il package. Rimuovo io ma non in questo branch

Reply:
@AndreaCimini90 confermo, è stato un mio refuso quando ho aggiunto il package. Rimuovo io ma non in questo branch

Reply:
@AndreaCimini90 @AndriBoca pullando da develop questo sarà risolto. Chiudo il commento